simulator/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM rust:1.70.0-bullseye
+
+RUN apt update && apt install -y cmake clang build-essential llvm
```

Comment:
We should specify versions for all of these to make the local build more deterministic.

Reply:
done